packages/SidePanel/src/components/Dialog/Dialog.js

```diff
@@ -6,6 +6,7 @@ const propTypes = {
   children: PropTypes.node.isRequired,
   footer: PropTypes.node,
   groupOffsetY: PropTypes.number,
+  hasBoxShadow: PropTypes.bool,
```

Comment:
seems unnecessarily specific, I'd go with `hasShadow`

Reply:
rip i forgot to tag WIP

Reply:
still confirming some styling stuff

Reply:
rip